pubnub-memberships/gradle.properties

```diff
@@ -1,4 +1,4 @@
 POM_ARTIFACT_ID=pubnub-memberships
 POM_NAME=pubnub-memberships
 POM_PACKAGING=jar
-VERSION_NAME=0.1.0
+VERSION_NAME=0.2.0
```

Comment:
Why we are changing version here. Aren't we changing version in release commit?

Reply:
This is a release commit